lib/pure/strformat.nim

```diff
@@ -211,6 +211,65 @@ The available floating point presentation types are:
 =================        ====================================================
 
 
+Limitations
+===========
+
+Because of the well defined order how templates and macros are
+expanded, strformat cannot expand template arguments.
+
+
+.. code-block:: nim
+  template myTemplate(arg: untyped): untyped =
+    echo "arg is: ", arg
+    echo &"--- {arg} ---"
+
+  let x = "abc"
+  myTemplate(x)
+
+First the template ``myTemplate`` is expanded, where every identifier
+``arg`` is substituted with it's argument. The ``arg`` inside the
```

Comment:
"it's" is a contraction of "it is" or of "it has", but the possessive determiner with the meaning "belonging to it" should be written as "its"

```suggestion
``arg`` is substituted with its argument. The ``arg`` inside the
```